CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.7.1 [2018.12.14]
+* Add `TextShow` instance for `ShortText` in `TextShow.Data.ShortText'
```

Comment:
Importantly, we should also mention that we:

1. Now depend on the `text-short` package.
2. Require `base-4.7` or later.